awesome-doc-code-sections.js

```diff
@@ -671,20 +671,19 @@ class CodeSection extends BasicCodeSection {
     static HTMLElement_name = 'awesome-doc-code-sections_code-section'
     static loading_animation = (function(){
     // TODO: loading_animation.* as opt-in, inline (raw github data) as fallback
-    // TODO: cache animation image (otherwise, too slow to load)
-        const loading_animation_fallback_url = 'https://github.com/GuillaumeDua/awesome-doc-code-sections/blob/main/images/loading_animation.gif?raw=true'
+        const loading_animation_fallback_url = 'url("https://raw.githubusercontent.com/GuillaumeDua/awesome-doc-code-sections/main/images/loading_animation.svg")'
         let loading_animation = document.createElement('div');
-        //  loading_animation.style.backgroundImage = `url("${loading_animation_fallback_url}")`
-        //  loading_animation.style.backgroundImage = 'url("loading_animation.svg")'
         Misc.apply_css(loading_animation, {
-            backgroundImage     : 'url("loading_animation.gif")',
+            // backgroundImage     : 'url("/images/loading_animation.svg")',
+            backgroundImage     : loading_animation_fallback_url,
```

Comment:
In my opinion its better and easier to, directly use an image instead of a background-image for your CSS